redis_json/src/c_api.rs

```diff
@@ -543,39 +579,24 @@ macro_rules! redis_json_module_export_shared_api {
             )
         }
 
-        static REDISJSON_GETAPI_V1: &str = concat!("RedisJSON_V1", "\0");
-        static REDISJSON_GETAPI_V2: &str = concat!("RedisJSON_V2", "\0");
-        static REDISJSON_GETAPI_V3: &str = concat!("RedisJSON_V3", "\0");
-        static REDISJSON_GETAPI_V4: &str = concat!("RedisJSON_V4", "\0");
+        // The apiname argument of export_shared_api should be a string literal with static lifetime
+        static mut VEC_EXPORT_SHARED_API_NAME : Vec<CString> = Vec::new();
```

Comment:
We can make it a static slice:
static VEC_EXPORT_SHARED_API_NAME: &'static str = &["RedisJSON_V1", ...];

Reply:
Also why does it need to be static lifetime?

Reply:
https://github.com/redis/redis/blob/unstable/src/module.c#L10498-L10500

> IMPORTANT: the apiname argument should be a string literal with static
> lifetime. The API relies on the fact that it will always be valid in
> the future.
>  int RM_ExportSharedAPI(RedisModuleCtx *ctx, const char *apiname, void *func) {